src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -2460,7 +2470,7 @@ namespace QgsWms
       {
         try
         {
-          QgsRectangle transformedBox = transform.transformBoundingBox( box );
+          const QgsRectangle transformedBox = transform.transformBoundingBox( box );
           box = transformedBox;
```

Comment:
```suggestion
          box = transform.transformBoundingBox( box );
```

Not sure it's worth the extra line :stuck_out_tongue_closed_eyes: